tests/test_bls.py

```diff
@@ -69,7 +69,8 @@ def test_hash_to_G2():
     message_hash = b'\x12' * 32
 
     domain_1 = 1
-    result_1 = hash_to_G2(message_hash, domain_1)
+    domain_in_bytes = domain_1.to_bytes(8, 'big')
```

Comment:
I think we can safely use little-endian here, so that we have the domain in little-endian everywhere.

Reply:
the spec still uses big-endian for this part -- i think the idea is to wait for the standardization before we change bls much more